### PR TITLE
Water Heater. Ability to send only mandatory parameters

### PR DESCRIPTION
**Pull request by VadymMelnychuk**
In my case I have command StartProgram that has 3 parameters ([Commands TestData Link](https://github.com/Andre0512/hon-test-data/blob/12a40ce2dc918e6d21bc1248ba917ad92826f018/test_data/wh_265/commands.json#L114)):
- onOffStatus
- machMode
- tempSel

But only `onOffStatus` is mandatory. 
The problem is when I send `StartProgram` command the body contains all 3 parameters with default values. As result I set `tempSel` to default value. Instead of this I want to send only mandatory parameters.
I'm also preparing changes Hon repository as well

P.S. It might fix issue with on/off AC too

### CODE REVIEW

**Comment by Andre0512** [comment]
Looks good so far, thanks!
You can also use black locally to format the code 🙂 
```pip install black; black pyhon/```

**Comment by VadymMelnychuk** [comment]
Thanks for help me with black :)
Now it's fixed

**Comment by VadymMelnychuk** [comment]
@Andre0512 
I added more options to manage the process of sending command

**Comment by Andre0512** [comment]
Good work, I think this can also resolve the new issue https://github.com/Andre0512/hon/issues/92 :slightly_smiling_face: 
To get the actions running, have a look at the [python-check.yml](https://github.com/Andre0512/pyhOn/blob/main/.github/workflows/python-check.yml). You can test the action local if you run:
```bash
pip install -r requirements_dev.txt
flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
flake8 . --count --exit-zero --max-complexity=10 --max-line-length=88 --statistics
mypy pyhon/
pylint  $(git ls-files '*.py')
black . --check```

**Review by Andre0512 — CHANGES_REQUESTED** [review]

File: pyhon/commands.py [hunk 1]
```diff
@@ -111,8 +119,10 @@ def _create_parameters(
             name = "program" if "PROGRAM" in self._category_name else "category"
             self._parameters[name] = HonParameterProgram(name, self, "custom")
 
-    async def send(self) -> bool:
-        params = self.parameter_groups.get("parameters", {})
+    async def send(self, onlyMandatory: bool = False) -> bool:
```
[the remark this answers is not in the document]
Comment (VadymMelnychuk) [reply pyhon/commands.py]: Done

**Review by Andre0512 — APPROVED** [review]

**Comment by Andre0512** [comment]
Thanks! It's now available in the new version `v0.15.0` :slightly_smiling_face:

**Pull request merged**